sql/src/test/java/org/apache/druid/sql/calcite/schema/SystemSchemaTest.java

```diff
@@ -330,10 +330,10 @@ public void testGetTableMap()
   public void testSegmentsTable() throws Exception
   {
     // total segments = 6
-    // segments 1,2,3 are published and available
-    // segments 4,5,6  are published but unavailable
-    // segment 3 is published but not served
-    // segment 2 is served by 2 servers, so num_replicas=2
+    // segments 1,2,3 are published but unavailable
```

Comment:
Can you adjust the comments here to refer to the segments being tested explicitly?

The 1,2,3 and 4,5,6 numbering confused me initially (I thought they were referring to `segment1`, `segment2`, etc. as defined earlier in the test).

Reply:
Yes, I see it can be confusing, changed the comment doc to make it more clear.